pkg/cassandra/config/config.go

```diff
@@ -144,23 +147,21 @@ func (c *Configuration) NewCluster() *gocql.ClusterConfig {
 			Password: c.Authenticator.Basic.Password,
 		}
 	}
+	tlsCfg, err := c.TLS.Config(logger)
+	if err != nil {
+		return nil, err
+	}
 	if c.TLS.Enabled {
 		cluster.SslOpts = &gocql.SslOptions{
-			Config: &tls.Config{
-				ServerName: c.TLS.ServerName,
-			},
-			CertPath:               c.TLS.CertPath,
```

Comment:
@yurishkuro instead of relying on the driver to load the certs I have changed it to use our TLS loading.